src/components/data/menuItems/DownloadMenuItem.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import { build } from "@cyverse-de/ui-lib";
+import { ListItemIcon, ListItemText, MenuItem } from "@material-ui/core";
+import { CloudDownload } from "@material-ui/icons";
+
+import ids from "../ids";
+import { useTranslation } from "i18n";
+
+function DeleteMenuItem(props) {
+    const { baseId, onClose, onDownloadSelected } = props;
+    const { t } = useTranslation("data");
+
+    return (
+        <MenuItem
+            key={build(baseId, ids.DOWNLOAD_MENU_ITEM)}
```

Comment:
This `key` is not needed here. Only `DeleteMenuItem` needs a `key` in the array in `RowDotMenu`.